traffic_ops/client/session.go

```diff
@@ -13,6 +13,7 @@
    limitations under the License.
 */
 
+// Package client provides Go bindings to the Traffic Ops RPC API.
```

Comment:
"RPC"?

Reply:
Remote Procedure Call (as opposed to a native Go API for example).